### PR TITLE
Update timeseries/timeseries_classification_from_scratch example

### PR DESCRIPTION
**Pull request by grasskin**
Updated to Keras3.

### CODE REVIEW

**Comment by grasskin** [comment]
Our validation accuracy is higher than it used to be (reading the loss image diff)

File: examples/timeseries/md/timeseries_classification_from_scratch.md [hunk 1]
```diff
@@ -177,15 +177,12 @@ model = make_model(input_shape=x_train.shape[1:])
 keras.utils.plot_model(model, show_shapes=True)
 ```
 
+<div class="k-default-codeblock">
+```
+You must install pydot (`pip install pydot`) for `plot_model` to work.
```
Comment (fchollet) [inline examples/timeseries/md/timeseries_classification_from_scratch.md:182]: Please install pydot and regenerate. This should show a model plot.

Comment (grasskin) [reply examples/timeseries/md/timeseries_classification_from_scratch.md]: Should be fixed now, also installed graphviz

**Review by fchollet — APPROVED** [review]
LGTM
